packages/playwright-ct-vue/package.json

```diff
@@ -1,8 +1,22 @@
 {
-    "name": "@playwright/ct-vue",
+    "name": "@playwright/experimental-ct-vue",
```

Comment:
The last time we prefixed a package with experimental it went wrong and people kept using it for years in blog articles (allure). Also they didn't known about it's successor.

Let's use the right name but a 0.x version which is more common?

Reply:
The problem is that we don't yet have the 'right name'.